e2e/nomostest/config_sync.go

```diff
@@ -87,8 +89,59 @@ var (
 	reconcilerPollingPeriod time.Duration
 	// hydrationPollingPeriod specifies hydration-controller polling period as time.Duration
 	hydrationPollingPeriod time.Duration
+
+	// DefaultRootReconcilerName is the root-reconciler name of the default RootSync object: "root-sync".
+	DefaultRootReconcilerName = core.RootReconcilerName(configsync.RootSyncName)
+	// DefaultRootRepoNamespacedName is the NamespacedName of the default RootSync object.
+	DefaultRootRepoNamespacedName = RootSyncNN(configsync.RootSyncName)
+	// DefaultRootSyncID is the ID of the default RootSync object.
+	DefaultRootSyncID = RootSyncID(configsync.RootSyncName)
 )
 
+// RootSyncNN returns the NamespacedName of the RootSync object.
+func RootSyncNN(name string) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: configsync.ControllerNamespace,
+		Name:      name,
+	}
+}
+
+// RepoSyncNN returns the NamespacedName of the RepoSync object.
+func RepoSyncNN(ns, name string) types.NamespacedName {
```

Comment:
nit: Is this function really needed?

Reply:
This already existed, I just moved it to the configsync file.

Reply:
FWIW, this trades off the ordering concern for being less to type and more consistently discoverable and named when together with RootSyncNN. Seems like a wash to me.